### ISSUE TITLE
Fix Tee's feet

### ISSUE THREAD

**Issue by android272** [teeworlds/teeworlds#1398]
In theatre you are taught to keep your body open. That is to always keep the front of your body facing as much to the audience as you can.  When two people speak with one another on stage, they need to face each other and the audience.  The way they accomplish this feat is to cheat out, halfway between profile and full front.  This position, depending on which side the actor is standing, is called one-quarter left or right.

In video games especially side scrollers such as teeworlds it is also true that the character should face one-quarter left, right or full front. This is not because we absolutely have to but because it is what the audience is used to and expects(whether they know it or not).  It is natural for the characters on screen/stage to be some what facing the audience.  If you take a look at other side scrollers  such as Mario, Sonic, Mega Man and others you will see that when their characters are standing still they are slightly turned toward the screen so that you can see more of the characters face and body and then when they walk they turn to a more natural walk.

If you look at a Tee's feet it looks as though they are actually turning their backs toward the audience. yet their eyes are looking as though they are turned one-quarter left or right.  The first thing that could be done to make Tees look better is to move their front foot(below would be the right foot) back and the back foot forward(below would be the left foot).  This will achieve the one-quarter look.

![feet position](https://cloud.githubusercontent.com/assets/5912527/9834959/5c216078-59a0-11e5-97fe-752f9129c1ca.png)

**Comment by TheRealOrangus** [comment]
Yes, sometimes it looks weird that a red team is attack blue team by their _"backs"_.
I agree, but partially: it should be fixed, but not as some kind of flip, maybe with some smoother way.
I like smoothness of animations in teeworlds.

**Comment by heinrich5991** [comment]
Currently weapon graphics aren't "smoothed" either. I guess it really depends on what you're used to.

**Comment by android272** [comment]
>Yes, sometimes it looks weird that a red team is attack blue team by their "backs".
I agree, but partially: it should be fixed, but not as some kind of flip, maybe with some smoother way.
I like smoothness of animations in teeworlds.

What do you mean by the smoothness of the animations in teeworlds?  This would be a simple fix of changing the x position of the feet.  In fact if someone were to point out where I might find the bit of code where Tees are drown I might make a patch myself.

**Comment by TheRealOrangus** [comment]
> Currently weapon graphics aren't "smoothed" either. I guess it really depends on what you're used to.

It is smoother than in mario at least, if that is not smooth, what it is for you?

> What do you mean by the smoothness of the animations in teeworlds? This would be a simple fix of changing the x position of the feet.

I mean that it would not be changed in a signle frame, some animation supposed to be there.
Like, the feet's X may smoothly "get swapped".

**Comment by TheRealOrangus** [comment]
> In fact if someone were to point out where I might find the bit of code where Tees are drown I might make a patch myself.

0.7: https://github.com/teeworlds/teeworlds/blob/master/src/game/client/render.cpp#L330
0.6: https://github.com/teeworlds/teeworlds/blob/0.6/src/game/client/render.cpp#L168

**Comment by android272** [comment]
Or... C is nothing like anything I have ever seen before and I have no idea what is going on here..  I get the logic of when things get drawn, but then after that the C++ syntax is just weird...
